### DISCUSSION TITLE
how to make Localization  for Predefined Roles?

### DISCUSSION THREAD

**Discussion by scil** [OrchardCMS/OrchardCore#10908] [Q&A]
like Administrator, Authenticated ... ( https://docs.orchardcore.net/en/dev/docs/reference/modules/Roles/)

**Comment by Skrypt** [comment]
Not everything can be translated yet. We have a Data Localization module that should cover these scenarios but it's not merged yet.

**Comment by hishamco** [comment]
We can localize the predefine role names, but as @Skrypt mentioned it will better to use Data localization which is still WIP in such case where data are changes

Hopefully, I will spend time on Data localization to make it ready to demo at least
